server/lease/lessor.go

```diff
@@ -359,6 +378,10 @@ func (le *lessor) Checkpoint(id LeaseID, remainingTTL int64) error {
 	return nil
 }
 
+func greaterOrEqual(first, second semver.Version) bool {
+	return first.Equal(second) || second.LessThan(first)
```

Comment:
Is it equivalent of: `!(second.LessThen(first))` or there is something subtle ?

Reply:
Nothing subtle, fixed.